Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/03-SyncUpForm/SyncUpForm-03-code-0003.swift

```diff
@@ -34,23 +34,12 @@ struct SyncUpForm {
       case .binding:
         return .none
 
-      case let .onDeleteAttendees(indexSet):
-        guard let firstDeletedIndex = indexSet.first
+      case let .onDeleteAttendees(indices):
```

Comment:
When you remove the last attendee in the list the app crashes. Previously an empty attendee was added - that's mentioned in the tutorial paragraph as well and it should be probably updated:
> We have also added some additional logic to make sure that the collection of attendees is not empty. This is the kind of logic we would like to get test coverage on eventually.

Reply:
The code in the TCA example works fine, you might want to reuse it:
```swift
case let .onDeleteAttendees(indices):
  state.syncUp.attendees.remove(atOffsets: indices)
  if state.syncUp.attendees.isEmpty {
    state.syncUp.attendees.append(
      Attendee(id: Attendee.ID())
    )
  }
  guard let firstIndex = indices.first
  else { return .none }
  let index = min(firstIndex, state.syncUp.attendees.count - 1)
  state.focus = .attendee(state.syncUp.attendees[index].id)
  return .none
```

Reply:
But I see that the focus works differently and the next chapter about testing is trying to cover the old behaviour. So the fix won't be so easy as I thought. Sorry 😅

Reply:
Yeah I need to take more time to look into this. The problem is that it is very hard for me to get a project that is in the exact state of this step in the tutorial. I have to start from the beginning, so it takes time.

Reply:
It seems to be very difficult to make changes and maintain these tutorials. I wanted to start use them too, but now I have to reconsider that intention.

Reply:
@tomassliz Do you mean writing your own DocC tutorials? Or working through our specific tutorial?

Reply:
@tomassliz I just pushed some fixes to this branch that I think simplifies things and makes things more correct.